Frends.AzureDataLake.UploadFiles/Frends.AzureDataLake.UploadFiles/Definitions/Result.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+
+namespace Frends.AzureDataLake.UploadFiles.Definitions;
+
+/// <summary>
+/// Task's result.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// Operation complete.
+    /// Operation is seens as completed if an ignorable error has occured and Options.ThrowErrorOnFailure is set to false.
+    /// </summary>
+    /// <example>true</example>
+    public bool Success { get; private set; }
+
+    /// <summary>
+    /// This object contains the source file path and the URL of the file.
```

Comment:
"... contains the list of local source file paths and their respective URLs in Azure Data Lake".

Reply:
+ I'll also add a little fix to return full url of file, not just container-relative path